content/en/docs/event-sources/kernel/_index.md

```diff
@@ -36,7 +36,7 @@ The classic {{< glossary_tooltip text="eBPF probe" term_id="ebpf-probe" >}} is a
 
 To install the eBPF probe, please refer to the [installation](/docs/getting-started/installation/#install-driver) page.
 
-To enable the eBPF support in Falco set the `FALCO_BPF_PROBE` environment variable to an empty value (ie. `FALCO_BPF_PROBE=""`, in this way Falco will search under `$HOME/.falco` directory for a file called `falco-bpf.o`) or otherwise explicitly set it to the path where the eBPF probe resides.
+To enable the eBPF support in Falco set the `engine.kind` configuration key to `ebpf` and eventually customize `engine.ebpf.probe`  to the path where the eBPF probe resides; the default path is the location used by `falcoctl driver` tool to install the eBPF probe.
```

Comment:
I have a suggestion for this: why don't we move the modern eBPF probe section up here, update the instructions to run it and move the classic/legacy one down? This way people with recent kernels will read relevant instruction first and have a better experience and only people that need eBPF but cannot run the new probe will follow the legacy instructions to run Falco on their system

Reply:
It's a great idea actually!
cc @leogr @Andreagit97 @incertum wdyt?

Reply:
totally agree

Reply:
Done! Let me know wdyt!